spec/ScoreSpec.js

```diff
@@ -0,0 +1,108 @@
+describe("Game", function() {
+  var score;
+  var frame;
+  var display;
+
+  beforeEach(function() {
+    game = new Game();
+    frame = new Frame();
+    display = new Display();
+  });
+
+  it("should have an initial score of 0", function() {
+    expect(game.score).toEqual(0);
+    expect(game.cumulatives).toEqual([]);
+  });
+
+  describe('Retrieving score from Frame', function() {
+    it('uses frame to get the score', function() {
```

Comment:
First, I would expect game.score probably to be a method, game.getScore(), since I think it's dynamic state dependent on other things. 

I would then expect the test here to look something more like:

```
game.getScore();
expect(frame.getScore).toHaveBeenCalledOnce();
```

Since that actually tests that it calls frame - what you are testing below would pass even if frame wasn't used and it calculate the score inside the game class.